apps/typed-dependency-tester/README.md

```diff
@@ -0,0 +1,17 @@
+# Typed Dependency Tester
+
+The Typed Depndency Tester is a tool used to detect if types in `MicrosoftTeams.d.ts` are importing types from other modules and giving them an implicit `any` resolution
```

Comment:
Nit: Spelling

```suggestion
The Typed Dependency Tester is a tool used to detect if types in `MicrosoftTeams.d.ts` are importing types from other modules and giving them an implicit `any` resolution
```

Also, are there other type-related errors this tool detects? It feels like it has detected other import/export-related issues.